src/main/java/io/gatling/mojo/EnterprisePackageMojo.java

```diff
@@ -137,6 +140,31 @@ private Set<Artifact> resolveTransitively(Artifact artifact) {
             .setProxies(session.getRequest().getProxies())
             .setLocalRepository(session.getLocalRepository())
             .setRemoteRepositories(session.getCurrentProject().getRemoteArtifactRepositories());
+
     return repository.resolve(request).getArtifacts();
   }
+
+  private static List<Artifact> getAllGatlingDeps(
+      Set<Artifact> allDeps, List<Artifact> depsWithGatlingGroupId) {
+    List<Artifact> depsWithGatlingHighchartsGroupId =
+        MojoUtils.findByGroupId(allDeps, GATLING_HIGHCHARTS_GROUP_ID);
+    List<Artifact> allGatlingDeps = new ArrayList<>();
+
+    allGatlingDeps.addAll(depsWithGatlingGroupId);
+    allGatlingDeps.addAll(depsWithGatlingHighchartsGroupId);
+
+    return allGatlingDeps;
+  }
+
+  private static Artifact getGatlingApp(List<Artifact> depsWithGatlingGroupId) {
```

Comment:
We don't really care about locating the gatling-app jar. We care about getting the Gatling version.